docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
 # Getting started
 
-TODO
+In this tutorial, we will download and install Pebble, configure layers, run the Pebble daemon, and work with layers and services. After this tutorial, you will have a basic understanding of what Pebble is, how to install it, and how to use it to orchestrate services. It takes about 15 minutes to complete.
+
+
+After that, you can continue exploring more advanced features and use cases (links at the end).
+
+## Prerequisites
+
+- A Linux machine.
+
+## Download and install Pebble
+
+Find the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following commands to download, extract, and install the latest release (replace `v1.11.0` with the latest tag and `amd64` with your architecture):
+
+```bash
+wget https://github.com/canonical/pebble/releases/download/v1.11.0/pebble_v1.11.0_linux_amd64.tar.gz
+tar zxvf pebble_v1.11.0_linux_amd64.tar.gz
+sudo mv pebble /usr/local/bin/ # make sure it's in your $PATH
+```
+
+## Verify Pebble installation
+
+Once the installation is complete, verify that `pebble` has been installed correctly by running:
+
+```bash
+pebble
+```
+
+This should produce output similar to the following:
+
+```bash
+Pebble lets you control services and perform management actions on
+the system that is running them.
+
+Usage: pebble <command> [<options>...]
+
+...
+```
+
+## Configure Pebble
+
+Now that Pebble has been installed, we can set up a basic configuration.
+
+First, let's create a directory for Pebble configuration:
+
+```bash
+mkdir -p ~/PEBBLE/layers
+export PEBBLE=$HOME/PEBBLE
+echo "export PEBBLE=$HOME/PEBBLE" >> ~/.bashrc # add $PEBBLE to your bashrc
+```
+
+Next, create a layer by running:
+
+```bash
+echo """\
+summary: Simple layer
+
+description: |
+    A simple layer.
+
+services:
+    http-server:
+        override: replace
+        summary: demo http server
+        command: python3 -m http.server 8080
+        startup: enabled
+""" > $PEBBLE/layers/001-http-server.yaml
+```
+
+This creates a simple layer containing only one service (which runs a basic HTTP server using Python's `http` module, listening on port 8080).
```

Comment:
```suggestion
This creates a simple layer containing only one service (`http-server`) which runs a basic HTTP server using Python's `http` module, listening on port 8080.
```

I suggest calling attention to the name of the service based on the layer configuration since at this point in time the reader doesn't have a full understanding of the structure of the YAML file.